.github/workflows/build-artifact.yaml

```diff
@@ -104,15 +104,20 @@ jobs:
           image: ${{ env.TT_METAL_DOCKER_IMAGE_TAG }}
           options: |
             --rm
+            --tmpfs /tmp
             -u ${{ env.RUNNER_UID }}:${{ env.RUNNER_GID }}
             -v ${{ github.workspace }}:${{ github.workspace }}
             -v /etc/passwd:/etc/passwd:ro
             -v /etc/shadow:/etc/shadow:ro
             -v /etc/bashrc:/etc/bashrc:ro
+            -v /mnt/MLPerf/ccache:/home/ubuntu/.ccache
```

Comment:
is `/mnt/MLPerf` mounted properly on all builders?

Reply:
and even if it's mounted, how are ensuring the mount is available when running?

Reply:
All the Ansible'd machines have it.  I haven't yet done the non-Ansible'd machines.

Reply:
I can add a check to verify.  IMHO we should fail the build if it's not there -- that's an indication that a build is in the build pool but hasn't been properly provisioned.  We should flag it so we can fix it.

Reply:
Even if it's installed, there's been times when accessing the mount still fails
For the models pipeline we restart the weka service before each job run and this has been the most reliable way to ensure weka connectivity so far

Reply:
Also, wouldn't multiple builders be sharing the same folder? Don't you need `sccache` for this type of sharing?

Reply:
> For the models pipeline we restart the weka service before each job run and this has been the most reliable way to ensure weka connectivity so far

You're suggesting we should force a remount of the drive at the start of the run?  Wouldn't that run afoul anything else running on that machine?  AFAIK some machines are shared.

Reply:
> Also, wouldn't multiple builders be sharing the same folder? Don't you need sccache for this type of sharing?

ccache can do it.  In the future I would like us to upgrade to a newer version and point ccache to a Redis server instead of the NFS drive.  But this works in the interim.

Reply:
> > For the models pipeline we restart the weka service before each job run and this has been the most reliable way to ensure weka connectivity so far
> 
> You're suggesting we should force a remount of the drive at the start of the run? Wouldn't that run afoul anything else running on that machine? AFAIK some machines are shared.

It would. That's why I think we should store the `ccache` artifacts locally for now and not rely on weka at all for building

Reply:
That's mostly defeating the purpose of having a cache.  😕

Reply:
It's either that, or we reduce the number of builders we have by getting rid of the doubling up on single machines.

We could also investigate seeing if using a new, separate mount for ccache is more reliable.

Reply:
In yesterday's meeting we agreed to use the network drive as-is and if we encounter issues we'll report them to the cloud team.  Longer term we may switch to Redis.  In the meantime each machine also has a local cache so will fallback gracefully should the network drive fall over.